app/cdap/components/ConnectionList/icons.js

```diff
@@ -54,3 +55,261 @@ export const GCSIcon = () => {
     </svg>
   );
 };
+
+export const InfoGraph = () => {
```

Comment:
here we need to have icon specific files in iconStore folder
i.e., inside ConnectionList folder create a new folder(name it as iconStore) inside this folder create icon specific file like GCSIcon.tsx, InfoGraphIcon.tsx